Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -737,7 +737,18 @@ bool CSystem::Init(const SSystemInitParams& startupParams)
 
     m_pCmdLine = new CCmdLine(startupParams.szSystemCmdLine);
 
-    AZCoreLogSink::Connect();
+    // Init AZCoreLogSink. Don't suppress system output if we're running as an editor-server
+    bool suppressSystemOutput = true;
+    if (const ICmdLineArg* isEditorServerArg = m_pCmdLine->FindArg(eCLAT_Pre, "editorsv_isDedicated"))
+    {
+        AZ::CVarFixedString lowercaseValue(isEditorServerArg->GetValue());
+        AZStd::to_lower(lowercaseValue.begin(), lowercaseValue.end());
+        if (lowercaseValue == "true")
```

Comment:
would be nice to add `GetBoolValue()` in addition to `GetFValue()` and `GetIValue()` into `ICmdLineArg` to avoid these manual string manipulations. 
Not sure if that class is being deprecated though since it's in the legacy CrySystem..

Reply:
Fixed! I made a little tweak to the other get methods and return a bool if a valid boolean was successfully found; the bool value itself is passed-by-reference and edited that way.  This way I could do something like...
bool myBoolArg;
if( cmdLineArg.GetBool(myBoolArg))
  // ... safely use myBoolArg

What do you think?